### DISCUSSION TITLE
bento-facebook and bento-twitter don't work anymore

### DISCUSSION THREAD

**Discussion by joomlarulez** [ampproject/bento#29] [Q&A] [answered]
Hi,

bento-facebook and bento-twitter don't work anymore

bento-facebook
https://bentojs.dev/components/bento-facebook/
it return a 404 error from google  

404. That’s an error.
The requested URL /2305221508000/frame.html was not found on this server. That’s all we know. 

 bento-twitter
 https://bentojs.dev/components/bento-twitter/
 it return a 404 error from google  

404. That’s an error.
The requested URL /2305221508000/frame.html was not found on this server. That’s all we know. 
 
It's like this for few weeks now

Wha't going on bento ?

Thanks for your support

**Comment by danielrozenberg** [comment]
We found the cause of the problem. I'm working on fixing it now - it should be a few hours

**Comment by joomlarulez** [comment]
Thanks :)

**Comment by danielrozenberg** [comment, marked as answer]
This should be resolved now!

**Comment by joomlarulez** [comment]
Thanks its' ok Now :)
